qt/scientific_interfaces/Indirect/test/InelasticDataManipulationElwinTabModelTest.h

```diff
@@ -0,0 +1,41 @@
+// Mantid Repository : https://github.com/mantidproject/mantid
+//
+// Copyright &copy; 2019 ISIS Rutherford Appleton Laboratory UKRI,
+//   NScD Oak Ridge National Laboratory, European Spallation Source,
+//   Institut Laue - Langevin & CSNS, Institute of High Energy Physics, CAS
+// SPDX - License - Identifier: GPL - 3.0 +
+#pragma once
+
+#include "IndirectDataValidationHelper.h"
+#include "InelasticDataManipulationElwinTabModel.h"
+#include "MantidAPI/AnalysisDataService.h"
+#include "MantidFrameworkTestHelpers/WorkspaceCreationHelper.h"
+#include <cxxtest/TestSuite.h>
+#include <gmock/gmock.h>
+
+using namespace MantidQt::CustomInterfaces;
+
+class InelasticDataManipulationElwinTabModelTest : public CxxTest::TestSuite {
```

Comment:
There seem to be multiple test paths missing here. Such as testing for background subtraction and normalisation, and the setup of the different algorithms (e.g. group)

Reply:
We cannot setup a test for the ElasticWindowMultiple algorithm since it is a python algorithm. I can make a test for the other algorithms

Reply:
Can the algorithm not be mocked so that you can check what properties are being set on it?